[BUG] python binary location

**Describe the bug**
Auto-responder triggers configured to run Python scripts (`.py`) fail to execute inside the Docker container. The application logs an error stating the script is "not found", even when the file exists and has correct permissions. This occurs because the application attempts to use `/usr/bin/python` as the interpreter, but the Docker container only provides `/usr/bin/python3`.

**To Reproduce**
Steps to reproduce the behavior:
1. Create a Python script (e.g., `test.py`) in the `/data/scripts/` directory mapped to the container.
2. Configure an Auto-Responder trigger with:
   - Trigger: `!test`
   - Response Type: `script`
   - Response: `/data/scripts/test.py`
3. Send a message matching the trigger (`!test`) to the node.
4. Check the container logs.
5. See error: `[ERROR] 🔧 Auto-responder script for "!test" not found: /data/scripts/test.py`

**Expected behavior**
The Python script should execute successfully using the available Python interpreter (`python3`) inside the container.

**Screenshots**
N/A

**Environment (please complete the following information):**
* Host OS: Linux (Raspberry Pi OS)
* Deployment type: Docker
  * Container version: `ghcr.io/yeraze/meshmonitor:latest`
* Database backend: SQLite

**Additional context**
The issue stems from `server/meshtasticManager.js` where the interpreter path is hardcoded for production environments.

around line 6524 (in `dist`):
```javascript
case 'py':
    interpreter = isDev ? 'python' : '/usr/bin/python';
    break;
```

The underlying error code is `ENOENT`, which the application interprets as "file not found", but in this case, it is the *interpreter* executable that is not found, not the script itself.

**Workaround verified:**
Running the following command to create a symlink fixes the issue immediately:
`docker exec -u root meshmonitor ln -s /usr/bin/python3 /usr/bin/python`

--- a/Dockerfile.armv7
+++ b/Dockerfile.armv7
@@ -60,6 +60,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     supervisor \
     gosu \
     && rm -rf /var/lib/apt/lists/* \
+    && ln -sf /usr/bin/python3 /usr/bin/python \
     && python3 -m venv /opt/apprise-venv \
     && /opt/apprise-venv/bin/pip install --no-cache-dir apprise "paho-mqtt<2.0"
 

--- a/src/server/meshtasticManager.ts
+++ b/src/server/meshtasticManager.ts
@@ -1604,7 +1604,7 @@ class MeshtasticManager {
 
     switch (ext) {
       case 'js': case 'mjs': interpreter = isDev ? 'node' : '/usr/local/bin/node'; break;
-      case 'py': interpreter = isDev ? 'python' : '/usr/bin/python'; break;
+      case 'py': interpreter = isDev ? 'python' : '/usr/bin/python3'; break;
       case 'sh': interpreter = isDev ? 'sh' : '/bin/sh'; break;
       default:
         this.updateGeofenceTriggerResult(trigger.id, 'error', `Unsupported script extension: ${ext}`);
@@ -1856,7 +1856,7 @@ class MeshtasticManager {
         interpreter = isDev ? 'node' : '/usr/local/bin/node';
         break;
       case 'py':
-        interpreter = isDev ? 'python' : '/usr/bin/python';
+        interpreter = isDev ? 'python' : '/usr/bin/python3';
         break;
       case 'sh':
         interpreter = isDev ? 'sh' : '/bin/sh';
@@ -7498,7 +7498,7 @@ class MeshtasticManager {
                 interpreter = isDev ? 'node' : '/usr/local/bin/node';
                 break;
               case 'py':
-                interpreter = isDev ? 'python' : '/usr/bin/python';
+                interpreter = isDev ? 'python' : '/usr/bin/python3';
                 break;
               case 'sh':
                 interpreter = isDev ? 'sh' : '/bin/sh';

--- a/src/server/server.ts
+++ b/src/server/server.ts
@@ -8309,7 +8309,7 @@ apiRouter.post('/scripts/test', requirePermission('settings', 'read'), async (re
         interpreter = isDev ? 'node' : '/usr/local/bin/node';
         break;
       case 'py':
-        interpreter = isDev ? 'python' : '/usr/bin/python';
+        interpreter = isDev ? 'python' : '/usr/bin/python3';
         break;
       case 'sh':
         interpreter = isDev ? 'sh' : '/bin/sh';

